src/verilog.y

```diff
@@ -5618,6 +5654,40 @@ vltOnFront<errcodeen>:
 			  if ($$ == V3ErrorCode::EC_ERROR) { $1->v3error("Unknown Error Code: "<<*$3<<endl);  } }
 	;
 
+vltDModuleE<strp>:
+		/* empty */				{ static string unit="__024unit"; $$=&unit; }
```

Comment:
Please space around =.  (Know this was probably cut-and-pasted from something old.)